avocado/core/nrunner_avocado_instrumented.py

```diff
@@ -51,7 +51,11 @@ def _run_avocado(runnable, queue):
         state = instance.get_state()
         # This should probably be done in a translator
         if 'status' in state:
-            state['status'] = state['status'].lower()
+            status = state['status'].lower()
```

Comment:
This is better now, but what I'm really missing is that `state['status']` already has something that is supposed to be a `result`, not a `status`. In line 55 you are checking if the `status` is something on the `results` list.

I can see that a translation is going one here, but would it be possible to avoid this? Maybe we need to change `state['status']` to `state['result']` too. Not sure.

Reply:
> This is better now, but what I'm really missing is that `state['status']` already has something that is supposed to be a `result`, not a `status`. In line 55 you are checking if the `status` is something on the `results` list.

Right, but that is generated by the avocado `INSTRUMENTED` test code, which if we change, we'll break the current runner.

> 
> I can see that a translation is going one here, but would it be possible to avoid this? Maybe we need to change `state['status']` to `state['result']` too. Not sure.

I can't see a way around this *now*, because of the compatibility with the current runner.

Reply:
Ok then. If its going to be fixed soon, no problem. Just to make sure that I had understood correctly.